content/docs/reference/common_attributes.md

```diff
@@ -0,0 +1,200 @@
+---
+$title: Common Attributes
+$order: 3
+---
+
+AMP provides a set of common attributes that are extended to many AMP components (and HTML elements).  This document describes each of the following attributes:
+
+* [fallback](#fallback)
+* [height](#height)
+* [heights](#heights)
+* [layout](#layout)
+* [media](#media)
+* [noloading](#noloading)
+* [on](#on)
+* [placeholder](#placeholder)
+* [sizes](#sizes)
+* [width](#width)
+
+## fallback
+
+The `fallback` attribute can be set on any HTML element, not just AMP elements. A fallback is a convention that allows the element to communicate to the reader that the browser does not support the element. If specified, a fallback element must be a direct child of the AMP element. The exact behavior with respect to the fallback is up to the element's implementation.
```

Comment:
to the end:, but typically the fallback element would be shown in place of the regular element.